packages/material-ui/src/ClickAwayListener/ClickAwayListener.js

```diff
@@ -18,6 +17,13 @@ function useMountedRef() {
   return mountedRef;
 }
 
+function mapEventPropToEvent(eventProp) {
+  if (typeof eventProp === 'string' && eventProp.startsWith('on')) {
```

Comment:
```suggestion
  if (typeof eventProp === 'string' && eventProp.indexOf('on') === 0) {
```

[Sorry](http://kangax.github.io/compat-table/es6/#test-String.prototype_methods_String.prototype.startsWith_a_href=_https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith_title=_MDN_documentation_img_src=_../mdn.png_alt=_MDN_(Mozilla_Development_Network)_logo_width=_15_height=_13_/_/a_nbsp;)